### PR TITLE
Rename all Initxxx to Projectxxx

### LINKED ISSUE #1286: Rename all `Initxxx` to `Projectxxx`
As we now have both init and download inside `InitResource`, `InitApplicationService`, maybe it would be better to rename all to `ProjectResource`, `ProjectApplicationService` and so on.

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #1286 

- folder **init** renamed to **project**
- folder **project** renamed to **tools**

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Plz hold on this PR
I need to rethink about what @hdurix suggested in his ticket

It's because I found a little bit strange the new context 'tools' which contains all the important business : managing projects

Tools should be used for toolings, like Maven, Gradle, NPM etc

**Comment by qmonmert** [comment]
@pascalgrimaud ok 🤝

**Comment by qmonmert** [comment]
@pascalgrimaud I renamed `project` folder to `tools` but maybe an other name will be better (it was an idea)

File: src/main/java/tech/jhipster/lite/generator/project/domain/ProjectDomainService.java [hunk 1]
```diff
@@ -39,10 +38,10 @@ public void init(Project project) {
   }
 
   @Override
-  public void addPackageJson(Project project) {
+  public void addPackageJson(tech.jhipster.lite.generator.tools.domain.Project project) {
```
Comment (MSaguer) [inline src/main/java/tech/jhipster/lite/generator/project/domain/ProjectDomainService.java:41]: Could this be simpified by adding an `import tech.jhipster.lite.generator.tools.domain.Project` ?

Comment (pascalgrimaud) [reply src/main/java/tech/jhipster/lite/generator/project/domain/ProjectDomainService.java:41]: thanks @MSaguer  but no need to review, I need to rethink about the scope of the initial ticket

**Comment by pascalgrimaud** [comment]
I'm closing this in favor of https://github.com/jhipster/jhipster-lite/pull/1458
Hope you're ok @qmonmert

**Comment by qmonmert** [comment]
@pascalgrimaud ok

**Pull request closed without merging**